api/src/main/resources/Reference_Application_Numeric_Concepts-26-2.x.xml

```diff
@@ -10,4 +10,4 @@
 <concept_numeric concept_id="5092" hi_absolute="100.0" low_absolute="0.0" units="%" allow_decimal="false"/>
 <concept_numeric concept_id="5242" hi_absolute="999.0" hi_critical="120.0" hi_normal="70.0" low_absolute="0.0" units="" allow_decimal="false"/>
```

Comment:
cc @dkayiwa  this is the file i meant: So i think with current numeric concepts  there were no changes with latter concepts, thats why it showed as if i didn't update them thanks